deno/fhir-embeddings/store.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from "fs";
+import "https://deno.land/std@0.192.0/dotenv/load.ts";
+import {
+  createClient,
+  SupabaseClient,
+} from "https://esm.sh/@supabase/supabase-js@2";
+
+import { Document } from "https://esm.sh/langchain/document";
+import { OpenAIEmbeddings } from "https://esm.sh/langchain/embeddings/openai";
+import { SupabaseVectorStore } from "https://esm.sh/langchain/vectorstores/supabase";
+
+function compileSearchParamsDocuments(searchParamsSpec: any): Document[] {
+  const docs: Document[] = [];
+  searchParamsSpec.entry.forEach((entry: any) => {
```

Comment:
`searchParamsSpec` will have an `entry` key? I wonder if it's necessary to guard it?

Reply:
alsooo, does `entry` have  type?